test/common/http/dispatcher_test.cc

```diff
@@ -73,16 +73,18 @@ TEST_F(DispatcherTest, SetDestinationCluster) {
   callbacks_called cc = {0, 0, 0, 0, 0, 0};
   bridge_callbacks.context = &cc;
   bridge_callbacks.on_headers = [](envoy_headers c_headers, bool end_stream,
-                                   void* context) -> void {
-    ASSERT_TRUE(end_stream);
+                                   void* context) -> void* {
+    EXPECT_TRUE(end_stream);
```

Comment:
What's the difference between `ASSERT_TRUE` and `EXPECT_TRUE`?

Reply:
`ASSERT_TRUE` aborts execution if the expectation fails. It also requires the enclosing function to have a `void` return type, because it actually calls `return`. `EXPECT_TRUE` does not abort execution, and has no requirements on the return-type of the enclosing function. Both will cause the test to fail if the passed assertion/expectation isn't met.